app/boards/shields/Kconfig.defconfig

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Please add the standard ZMK header here, e.g. 

```
# Copyright (c) 2020 The ZMK Contributors
# SPDX-License-Identifier: MIT
```

Reply:
added